docs/how_to_guides/how_to_use_parameter_sweep_monte_carlo.rst

```diff
@@ -0,0 +1,116 @@
+Monte Carlo testing with the Parameter Sweep
+============================================
+
+Overview
+--------
+
+This guide introduces two important features of the parameter sweep tool: (1) the ability to prescribe random model values and (2) the ability to take advantage of parallel computing resources for large runs.  It is recommended that new users start with the :ref:`beginner parameter sweep guide<how_to_use_parameter_sweep>` before proceeding.
+
+.. shows you how to use the parameter sweep tool to explore the effect of changing model parameters or decision variables within your WaterTAP model.
+
+.. This might be useful, for example, if you have an existing model of a multi-stage treatment train and you'd like to see the effect of varying Pump 1 pressure and Pump 2 pressure independently (where all possible combinations of Pump 1 and Pump 2 pressure will be explicitly tested).
+.. The type and quantity of parameters to be varied are easily changed following steps like the ones below.
+
+How To
+------
+
+As before, we begin by importing or explicitly programming any functions relating to flowsheet building/specification, simulation, and optimization setup steps.  We'll use the same RO with energy recovery flowsheet for this example.
+
+.. testsetup::
+
+    # quiet idaes logs
+    import idaes.logger as idaeslogger
+    idaeslogger.getLogger('ideas.core').setLevel('CRITICAL')
+    idaeslogger.getLogger('ideas.core.util.scaling').setLevel('CRITICAL')
+    idaeslogger.getLogger('idaes.init').setLevel('CRITICAL')
+
+.. testcode::
+
+    # replace this with your own flowsheet module, e.g.
+    # import my_flowsheet_module as mfm
+    import watertap.examples.flowsheets.RO_with_energy_recovery.RO_with_energy_recovery as RO_flowsheet
+
+Once this is done, import the parameter sweep tool and two different random sampling classes
+
+.. testcode::
+
+    from watertap.tools.parameter_sweep import parameter_sweep, UniformSample, NormalSample
+
+The parameter sweep tool currently offers three random classes:
+
+* ``UniformSample``: Draw random samples uniformly from an upper and lower range.
+* ``NormalSample``: Draw samples from a normal distribution given a mean and standard deviation.
+* ``LatinHypercubeSample``: Draw samples using a Latin Hypercube algorithm which may yield a more complete exploration of high-dimensional parameter spaces. Note that currently this sample type may not be combined with other sampling types.
```

Comment:
Note that, currently,...

Reply:
I'm not really sure this comma is necessarily, and where not needed for clarity I'm inclined to leave them out.
https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Usage/faq0314.html